iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -734,6 +736,40 @@ static int addUserPropertiesTouMqttMessage(IOTHUB_MESSAGE_HANDLE iothub_message_
     return result;
 }
 
+#ifdef RUN_SFC_TESTS
+//
+// isMqttMessageSfcType checks to see if the message is a service-fault-control message.
+//
+static bool isMqttMessageSfcType(IOTHUB_MESSAGE_HANDLE iothub_message_handle)
+{
+    bool result = false;
+    const char* const* propertyKeys;
+    const char* const* propertyValues;
+    size_t propertyCount;
+    size_t index;
+    MAP_HANDLE properties_map = IoTHubMessage_Properties(iothub_message_handle);
+    if (properties_map != NULL)
+    {
+        if (Map_GetInternals(properties_map, &propertyKeys, &propertyValues, &propertyCount) != MAP_OK)
+        {
+            LogError("Failed to get the internals of the property map.");
+        }
+        else
+        {
+            for (index = 0; index < propertyCount && result == 0; index++)
```

Comment:
not needed with the `break` below
```suggestion
            for (index = 0; index < propertyCount; index++)
```